OpenRA.Mods.Common/Traits/Render/LeavesTrails.cs

```diff
@@ -125,7 +125,7 @@ public void Tick(Actor self)
 				cachedFacing = facing != null ? facing.Facing : 0;
 				ticks = 0;
 
-				cachedInterval = isMoving && !wasStationary ? Info.MovingInterval : Info.StationaryInterval;
+				cachedInterval = isMoving ? Info.MovingInterval : Info.StationaryInterval;
```

Comment:
Ah yes, I see.  Wouldn't it make sense to reset ticks to 0 when transitioning from stationary to moving?